CONTRIBUTING.MD

```diff
@@ -0,0 +1,11 @@
+# How to contribute
+
+**Nya Chan** is constantly evolving as we need more and more features for Discord. Everyone is welcomed to contribute !
+
+## Getting started
+ * Make sure you have a [GitHub account](https://github.com/signup/free)
+ * Install [git-flow](https://danielkummer.github.io/git-flow-cheatsheet/#setup)
```

Comment:
don't enforce this.   git-flow I mean. It's not necessary you just need to describe the submission process.